env_osx.sh

```diff
@@ -1,9 +1,32 @@
 #!/bin/sh
+#
+# The following optional environment variables can be set to configure the build:
+#  - CURA_TARGET_OSX_VERSION: The minimum OSX version you are targeting
+#  - CURA_OSX_SDK_VERSION   : The OSX SDK version to use for compiling
+#
 
-export MACOSX_DEPLOYMENT_TARGET=10.7
-export CC=/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang
-export CXX=/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang++
-export CPATH="/usr/local/opt/openssl/include:/usr/local/opt/sqlite/include:$CPATH"
-export LIBRARY_PATH="/usr/local/opt/openssl/lib:/usr/local/opt/sqlite/lib:$LIBRARY_PATH"
-export PKG_CONFIG_PATH="/usr/local/opt/openssl/lib/pkgconfig:/usr/local/opt/sqlite/lib/pkgconfig:$PKG_CONFIG_PATH"
+
+# (Optional) Minimum OSX version for deployment
+if [ -z "${CURA_TARGET_OSX_VERSION}" ]; then
+    echo "Using ${CURA_TARGET_OSX_VERSION} as the minimum OSX version."
```

Comment:
Shouldn't that `if` statement be using `-n`? It currently checks if `${CURA_TARGET_OSX_VERSION}` is a zero-length string, and if it's zero length proceeds to use it everywhere.

Reply:
Fixed in 1a847a29ede41bd44c91785dbd5236da4997693e